cmd/access/node_builder/access_node_builder.go

```diff
@@ -1422,6 +1435,8 @@ func (builder *FlowAccessNodeBuilder) Build() (cmd.Node, error) {
 		}).
 		Module("backend script executor", func(node *cmd.NodeConfig) error {
 			builder.ScriptExecutor = backend.NewScriptExecutor()
+			builder.ScriptExecutor.SetMinCompatibleHeight(builder.scriptExecMinBlock)
+			builder.ScriptExecutor.SetMaxCompatibleHeight(builder.scriptExecMaxBlock)
```

Comment:
we can initialize the script executor with the height range? and also log the actual height range?

Reply:
addressed in [2134775](https://github.com/onflow/flow-go/pull/5283/commits/21347755c21a534794ddf2a73d7d54c7bac559b6)